lms/static/scripts/frontend_apps/config.js

```diff
@@ -84,6 +88,7 @@ import { createContext } from 'preact';
  * @prop {string|null} authUrl
  * @prop {boolean} invalidScope
  * @prop {string} errorDetails
+ * @prop {OAuthErrorCode|null} errorCode
```

Comment:
As noted above, if the default is `null` then the initialization for `errorCode` in `OAuth2RedirectErrorApp` should either be `null` or just omit it if we know the backend is always going to set it.